src/bottom-sheet/bottom-sheet.ts

```diff
@@ -0,0 +1,210 @@
+import { html, LitElement, PropertyValues, nothing } from 'lit';
```

Comment:
I think it is best to `tap-bottom-sheet` default slot should be its body content.